Sources/LocalNetworkConnectivity/RemoteNetworkDataSource.swift

```diff
@@ -12,8 +12,8 @@
 import Foundation
 
 enum RemoteNetworkCellType: Int {
-    case cloud = 0, wifi
-    static let count: Int = 2
+    case cloud = 0, streaming, wifi
+    static let count: Int = 3
```

Comment:
Is the `cloud = 0`, required? 
I think enum automatically assigns the initial case's value to `0` :D

Reply:
It's not automatically 0 from the documentation "Unlike C and Objective-C, Swift enumeration cases are not assigned a default integer value when they are created"

Reply:
Right! I think that only applies to when the enum's case has not been declared. If the type is declared,
> When you’re working with enumerations that store integer or string raw values, you don’t have to explicitly assign a raw value for each case. When you don’t, Swift will automatically assign the values for you.